src/flamegraph/mod.rs

```diff
@@ -424,22 +460,26 @@ fn filled_rectangle<W: Write>(
     y1: usize,
     y2: usize,
     color: (u8, u8, u8),
+    svg_event_empty: &mut Event,
```

Comment:
There are so many arguments here now. Maybe make them a struct?

Reply:
I picked up the idea, the code looks better indeed.